packages/drizzledb-pg/package.json

```diff
@@ -11,9 +11,9 @@
   },
   "scripts": {
     "clean": "del .turbo coverage",
-    "db:generate": "drizzle-kit generate:pg",
-    "db:migrate": "node --env-file=.env --import=tsx ./src/migrate.ts",
-    "db:push": "drizzle-kit push:pg",
+    "db:generate": "node --import=tsx ./node_modules/drizzle-kit/bin.cjs generate:pg",
```

Comment:
In order to allow Drizzle ORM to process the typescript files without the need to transpile to javascript. Need to run the underlying bin files using TSX instead.